vault/identity_store.go

```diff
@@ -258,12 +278,12 @@ func (i *IdentityStore) entityByAliasFactors(mountAccessor, aliasName string, cl
 		return nil, nil
 	}
 
-	return i.MemDBEntityByAliasID(alias.ID, clone)
+	return i.MemDBEntityByAliasIDInTxn(txn, alias.ID, clone)
 }
 
 // CreateEntity creates a new entity. This is used by core to
```

Comment:
Don't forget to update this comment :) Small nit, but I prefer `CreateOrFetchEntity`.

Reply:
Done.